bundle/libraries/local_path_test.go

```diff
@@ -57,6 +57,7 @@ func TestIsLibraryLocal(t *testing.T) {
 		{path: "s3://mybucket/path/to/package", expected: false},
 		{path: "dbfs:/mnt/path/to/package", expected: false},
 		{path: "beautifulsoup4", expected: false},
+		{path: "-e some/local/path", expected: false},
```

Comment:
There are some more tricky cases that we should cover here :-/ AI might help identify and handle them

```
-i http://myindexurl.com
--trusted-host pypi.org
--platform win_amd64
-e ${workspace.file_path}
-e /Workspace/path/to
-e /Volumes/path/to
```

Btw, Andrew, please consider how to incorporate such cases. It could be that what you have here is just net-better than what we have now and that we should have a followup with refinements. It could also be that we just need to revise this PR to cover things like `-i` properly, also to make sure we don't regress anything.

Reply:
We have quite a good coverage for this overall, but I've added some additional cases for unit / acceptacne tests